README.md

```diff
@@ -1,6 +1,52 @@
+# Welcome to Ansible Examples!
 
-Ansible Examples
-----------------
+First and foremost, for those who don't know, [Ansible](https://www.ansible.com)
+is a radically simple IT automation platform that makes your applications and
+systems easier to deploy. It allows you and your team to avoid writing scripts
+or custom code to deploy and update your applications, systems, and various
+classifications of network attached devices. Ansible allows you to automate in a
+language that approaches plain English, with no agents to install on remote
+systems and use native protocols based on device type such as SSH for
+UNIX-style operating systems, WinRM for Windows systems, REST APIs (httpapi)
+for REST API appliances, and many many more. Ansible is capable of speaking to
+any native protocol to any device.
 
-This repository contains examples and best practices for building Ansible Playbooks.
+If you are not familiar with Ansible and have arrived here looking for a place
+to start, please check out the [Getting
+Started
+Guide](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html)
+first.
+
+Now that's out of the way, let's dive right in.
+
+# Ansible Examples
+
+This repository contains examples and recommended practices for creating
+[Ansible
+Playbooks](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html).
+
+Here you will find various examples of common use cases that a modern [Systems
+Administrator](https://en.wikipedia.org/wiki/System_administrator) or
+[DevOps](https://en.wikipedia.org/wiki/DevOps) Practitioner may find themselves
+needing to automate. The goal of this repository and its contents are to provide
+*examples* that are functional and offer recommended guideance and how to
+accomplish certain tasks using common patterns. This repository is not
+inherently meant to provide "Production Ready" automation workflows, even though
+the content might be of that quality, it's certainly not a goal.
```

Comment:
```suggestion
the content might be of that quality, but rather to give you a good starting place to work from.
```
YMMV, I'd rather tell folks at least one thing that this *is* for, along with the things it is not for.